packages/core/src/App/app.jsx

```diff
@@ -47,11 +47,14 @@ const AppWithoutTranslation = ({ root_store }) => {
             import('@deriv/deriv-charts/dist/smartcharts.css');
         };
 
-        const loadExternalScripts = () => {
-            // Load external scripts once the app is fully loaded
-            setTimeout(() => {
-                initHotjar(root_store.client);
-            }, 5000);
+        const loadExternalScripts = async () => {
+            const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
why is it a promise instead of just a regular timeout ?

Reply:
he is probably avoiding nested timeout or counting the timer in cummulative manner (delay1, delay1 + delay2) since there are 2 things that need to be delayed one after another.

Reply:
Yeah, I don't want to add multiple concurrent timeout in case if we keep adding more things to this in the future